pkg/model/components/etcdmanager/model.go

```diff
@@ -558,7 +564,7 @@ func (b *EtcdManagerBuilder) buildPod(etcdCluster kops.EtcdClusterSpec, instance
 	}
 
 	{
-		container.Command = exec.WithTee("/etcd-manager", args, "/var/log/etcd.log")
+		container.Command = exec.WithTee("/ko-app/etcd-manager", args, "/var/log/etcd.log")
```

Comment:
I wonder if we should use `alsologtostderr` and `logfile` instead of the tee "hack"

Reply:
The goal would be to just override args, and rely on Entrypoint being set correctly

Reply:
That would be nice, if possible.